src/shellstrings.c

```diff
@@ -0,0 +1,105 @@
+
+/* 
+ * String output functions to be used in the shell
+ *
+ * See shellstrings.h for function documentation.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "shellstrings.h"
+
+/* See shellstrings.h */
+void shell_intro() {
+    printf("------- Welcome to Spellcheck! ------\n\n");
+
+    printf("Please load a file to begin. Additionally, select an output mode and/or choose dictionary before running the program.\n\n");
+
+    printf("f [/path/file.txt]           : open file\n");
+    printf("d [/path/dictionary.txt]     : input custom dictionary file\n");
+    printf("m [number]                   : switch output mode - quiet (1) verbose (2) or interactive (3) \n");
+    printf("h                            : help\n");
+    printf("q                            : quit program\n");
+}
+
+/* See shellstrings.h */
+void shell_save() {
+    printf("p : print all changes\n");
+    printf("s : save to existing file\n");
+    printf("c : save to custom file\n");
+    printf("r : re-edit file\n");
+    printf("q : quit program\n\n");
+}
+
+/* See shellstrings.h */
+void shell_print(char** lines) {
+    int i = 0;
+
+    printf("\n\n");
+    while (lines[i] != NULL) {
+        printf("%s", lines[i]);
+        i++;
+    }
+    printf("\n\n");
+}
+
+
+/* See shellstrings.h */
+void shell_error(char* error_text) {
+	printf("=> ERROR: %s <=\n", error_text);
+}
+
+/* See shellstrings.h */
+void shell_usage() {
+    char *u =
+    "Please use designated format:\n\n"
+    "./spellcheck [-flag] [~/path/file.txt] [-saveflag] [~/path/destination.txt] \n\n"
+    "For help, run without command line arguments and enter help page\n";
```

Comment:
This is unnecessary, please remove as well.

Reply:
done!